core/include/scipp/core/string.h

```diff
@@ -81,13 +92,17 @@ template <class T> std::string array_to_string(const T &arr) {
       s += "..., ";
       i = size - 2;
     }
-    s += element_to_string(arr[i]);
+    s += element_to_string(arr[i], unit);
   }
   s.resize(s.size() - 2);
   s += "]";
   return s;
 }
 
+template <class T> std::string array_to_string(const T &arr) {
+  return array_to_string(arr, std::string(""));
+}
```

Comment:
Can you avoid this wrapper and also the wrapper `element_to_string` above by making the empty string a default argument?

Reply:
good suggestion. thanks